pegasus/sites.v3/code.org/views/census_map.haml

```diff
@@ -1,7 +1,15 @@
 %link{rel: "stylesheet", type: "text/css", href: "/css/census_map.css"}
 -max_height_percentage = 0.65 if max_height_percentage.nil?
 
-#gmap.full-width
+#map
+  #gmap.full-width
+  #map-footer
+    %right{style: 'float: right;'}
+      %span{style: "font-family: 'Gotham 7r', sans-serif; font-size: 20px; margin: 0 25px 0 25px;"} in partnership with
+      %img{src: '/images/fit-300/avatars/computer_science_teachers_association.png'}
```

Comment:
Just shared with Marina and Miranda. Can we do style touch ups?

- Make the partner logo fit-200 instead
- Add 10px space between map and partner

![image](https://user-images.githubusercontent.com/1920530/30302551-96bef474-9716-11e7-9a40-35173bc6d965.png)